x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ilm/TimeseriesLifecycleType.java

```diff
@@ -70,16 +68,16 @@ public class TimeseriesLifecycleType implements LifecycleType {
         ShrinkAction.NAME,
         ForceMergeAction.NAME
     );
-    public static final List<String> ORDERED_VALID_COLD_ACTIONS = Stream.of(
+    public static final List<String> ORDERED_VALID_COLD_ACTIONS = Arrays.asList(
         SetPriorityAction.NAME,
         UnfollowAction.NAME,
         ReadOnlyAction.NAME,
         SearchableSnapshotAction.NAME,
         AllocateAction.NAME,
         MigrateAction.NAME,
         FreezeAction.NAME,
-        IndexSettings.isTimeSeriesModeEnabled() ? RollupILMAction.NAME : null
-    ).filter(Objects::nonNull).toList();
+        RollupILMAction.NAME
```

Comment:
Please remove `RollupILMAction` completely from here and I will enable it in https://github.com/elastic/elasticsearch/pull/87269